src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c

```diff
@@ -30,25 +30,110 @@ ActorInit En_Snowwd_InitVars = {
     (ActorFunc)EnSnowwd_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80AF7A90 = {
-    { COLTYPE_TREE, AT_NONE, AC_ON | AC_HARD | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK5, { 0x00000000, 0x00, 0x00 }, { 0x0100020A, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_TREE,
+        AT_NONE,
+        AC_ON | AC_HARD | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK5,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x0100020A, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 18, 60, 0, { 0, 0, 0 } },
 };
 
-#endif
+s32 D_80AF7ABC[] = { 0x00000000, 0x00000000, 0x00000000 };
 
-extern ColliderCylinderInit D_80AF7A90;
+s32 D_80AF7AC8[] = { 0x00000000, 0xC0800000, 0x00000000 };
+
+s32 D_80AF7AD4[] = { 0xFFFFFFFF };
+
+s32 D_80AF7AD8[] = { 0xC8C8DC00, 0x00000000 };
 
 extern UNK_TYPE D_06001AA0;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Snowwd/EnSnowwd_Init.s")
+void EnSnowwd_Init(Actor* thisx, PlayState* play) {
+    EnSnowwd* this = THIS;
+
+    this->actor.home.rot.z = 0;
+    this->actor.home.rot.y = 0;
+    this->unk190 = 0;
+    this->actor.uncullZoneForward = 4000.0f;
+    this->actor.uncullZoneScale = 2000.0f;
+    this->actor.uncullZoneDownward = 2400.0f;
+    Collider_InitAndSetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = func_80AF76F0;
+}
+
+void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
+    EnSnowwd* this = THIS;
+
+    Collider_DestroyCylinder(play, &this->collider);
+}
+
+void func_80AF76F0(EnSnowwd* this, PlayState* play) {
+    s32 pad;
+    Actor* thisx = &this->actor;
+    f32 sp54;
+    Vec3f sp48;
+
+    if (this->collider.base.acFlags & 2) {
+        this->collider.base.acFlags = this->collider.base.acFlags & 0xFFFD;
```

Comment:
```suggestion
    if (this->collider.base.acFlags & AC_HIT) {
        this->collider.base.acFlags &= ~AC_HIT;
```
There's a macro for this. Also, `this->collider.base.acFlags = this->collider.base.acFlags & ` is the same as `this->collider.base.acFlags &=`. You'll also learn to recognize this as "unsetting a bit", which usually takes the form of `&= ~`. The number `0xFFFD` can be written as `~2`